.github/workflows/on-release.yml

```
@@ -124,10 +124,12 @@ jobs:
     runs-on: ubuntu-22.04
     permissions:
       id-token: write
-      contents: read
+      contents: write
     steps:
       - name: Checkout
         uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
 
       - name: Remove temporary branch
+        env:
+          WORKING_BRANCH: ${{needs.complete-release-branch-transaction.outputs.WORKING_BRANCH}}
         run: git push origin --delete "${WORKING_BRANCH}"
```

Looks like I could simply inline this - should I?

---

Yes, you can just do
```suggestion
        run: git push origin --delete "${{ needs.complete-release-branch-transaction.outputs.WORKING_BRANCH }}"
```

---

Ugh, missed your comment. Let me add that to my branch with tiny fixes :)